dev/io.openliberty.org.apache.myfaces.3.0/bnd.bnd

```diff
@@ -0,0 +1,128 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-SymbolicName: io.openliberty.org.apache.myfaces.3.0
+
+-fixupmessages.missingexport: "Used bundle version * for exported package";is:=ignore
+
+instrument.ffdc: false
+instrument.classesExcludes: \
+  org/**/*.class
+
+
+Service-Component: \
+  com.ibm.ws.faces_3_0.taglib; \
+    implementation:=com.ibm.ws.faces.taglibconfig.FacesGlobalTagLibConfig; \
+    provide:='com.ibm.wsspi.jsp.taglib.config.GlobalTagLibConfig'; \
+    properties:="service.vendor=IBM", \
+
+Include-Resource: \
+  @${repo;org.apache.myfaces.core.impl;3.0.0.RC1;EXACT}!/META-INF/**, \
+  @${repo;io.openliberty.jakarta.faces.3.0}!/META-INF/resources/**, \
+  @${repo;io.openliberty.jakarta.faces.3.0}!/META-INF/internal-resources/**, \
+  META-INF=@src/META-INF
+
+DynamicImport-Package: \
+  com.ibm.ws.jsf.cdi, \
+  com.ibm.ws.jsf.config.annotion,\
+  com.ibm.ws.jsf.config.resource,\
+  com.ibm.ws.jsf.ee, \
+  com.ibm.ws.jsf.spi.impl, \
+  com.ibm.ws.jsf.extprocessor
+
+
+# Don't export the org.apache.myfaces.buildtools package as that is only needed for compilation.  The jar that contains the
+# org.apache.myfaces.buildtools package is myfaces-builder-annotations-1.0.9.jar
+# Additionally, a subset of these org.apache.myfaces packages are exported by the thirdparty.faces-3.0 bundle - so we exclude them here.
+Export-Package: \
+  !org.apache.myfaces.buildtools.*, \
+  !org.apache.myfaces.renderkit.html, \
+  !org.apache.myfaces.shared.config, \
+  !org.apache.myfaces.shared.renderkit, \
+  !org.apache.myfaces.shared.renderkit.html, \
+  !org.apache.myfaces.shared.renderkit.html.util, \
+  com.ibm.ws.faces.taglibconfig, \
+  org.apache.myfaces.application;thread-context=true;version=3.0.0, \
```

Comment:
It would be nice to define an implementation version variable here for the export version so when we upgrade to the newer version we only have to change it in one spot.

Reply:
I agree. However, I ran into an issue where imports/exports didn't match because of 3.0.0 / 3.0.0-RC1 versioning. That mismatch cause some problems for me. I can test again, however. Once we are on a final version, I can switch it over to use an implementation version variable.

Reply:
I can just make two variables as a workaround.

Reply:
https://github.com/OpenLiberty/open-liberty/issues/13889